great-idea/index.html

```diff
@@ -17,7 +17,6 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
```

Comment:
Also, I am able to just leave comments too if it is something I don't want you to have to approve or a change and I just want to make a general comment.